### DISCUSSION TITLE
Shadow Banned From GitHub Trending page?

### DISCUSSION THREAD

**Discussion by ChrisCavs** [community/community#57196] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

It would appear, based on the JavaScript Trending page, that I have been shadow banned on GitHub.

Every day for the past month, my recent repository [aimless.js](https://github.com/ChrisCavs/aimless.js) has received more stars than most of the currently trending JavaScript repositories, but has never appeared as trending.  Similarly, past projects of mine such as [rallax.js](https://github.com/ChrisCavs/rallax.js) performed well but never appeared as trending.  No repository of mine has ever appeared on the trending page.

This leads to the question: is GitHub's algorithm for what's trending really, really, really bad.  Or, is my account shadow banned?

Either way, I believe this is something that needs to be discussed.

Examples of trending JS repositories at the time of this post:
* [safak / nextjs-tutorial](https://github.com/safak/nextjs-tutorial) received just 9 stars today
* [A-d-i-t-h-y-a-n / hermit-md](https://github.com/A-d-i-t-h-y-a-n/hermit-md) received just 2 stars
* [soumyajit4419 / Portfolio](https://github.com/soumyajit4419/Portfolio) received just 5 stars
* [TusharKesarwani / Front-End-Projects](https://github.com/TusharKesarwani/Front-End-Projects) received just 3 stars

My library [aimless.js](https://github.com/ChrisCavs/aimless.js) received 150+ stars in the last 3 days.

**Please Note**:
I do not wish to say that my projects are any more important than other other project.  I wish solely to point out the obvious flaw in GitHub's algorithm that would lead to this occurring.  These flaws actively harm the community of developers that enjoy participating in open source software, and vastly limit the exposure of small-time developers.

**Comment by dmytrostriletskyi** [comment]
Hi, @ChrisCavs.

I have the same problem with being getting into the trending. 
By any chance, have you figured the algorithm out? 

Thanks in advance,
Dmytro

**Comment by diyaryvzz** [comment]
GitHub's trending algorithm takes into account a number of factors, including:

- Star count: The number of stars for a repository is used to measure the popularity of the repository.
- Number of followers: The number of followers of a repository is used to measure the interest of the repository.
- Last activity: The last activity of a repository is used to measure the timeliness of the repository.
- Language: The language of a repository is used to measure the audience of the repository.
- Popular tags: A repository's tags are used to measure the repository's interests.
- User's past activity: A user's past activity is used to measure the user's interests.

These factors are weighted using an algorithm to determine whether a repository is trending. Once it is decided whether the repository is trending or not, the repository is shown on the trends page.
